prepare.ipynb

```diff
@@ -704,7 +704,7 @@
    "source": [
     "## Interpolate\n",
     "\n",
-    "Interpolate the data onto a regular grid at constant height. Height will be set to 500 m to avoid downward continuing data. To do so, we'll use the `hm.EquivalentSourcesGB` class from Harmonica. The `window_size` is set to limit the amount of RAM used in the interpolation to about 5 Gb. If you have less memory available, decrease the window size."
+    "Interpolate the data onto a regular grid at constant height. Height will be set to 500 m to avoid downward continuing data. To do so, we'll use the[`hm.EquivalentSourcesGB`](https://www.fatiando.org/harmonica/dev/user_guide/equivalent_sources/gradient-boosted-eqs.html)  class from Harmonica. The `window_size` is set to limit the amount of RAM used in the interpolation to about 5 Gb. If you have less memory available, decrease the window size."
```

Comment:
```suggestion
    "Interpolate the data onto a regular grid at constant height. Height will be set to 500 m to avoid downward continuing data. To do so, we'll use the [`hm.EquivalentSourcesGB`](https://www.fatiando.org/harmonica/dev/user_guide/equivalent_sources/gradient-boosted-eqs.html)  class from Harmonica. The `window_size` is set to limit the amount of RAM used in the interpolation to about 5 Gb. If you have less memory available, decrease the window size."
```